src/Hyperwallet.js

```diff
@@ -94,6 +94,106 @@ export default class Hyperwallet {
         this.client.doGet("users", options, Hyperwallet.handle204Response(callback));
     }
 
+    /**
+     * Activate a user
+     *
+     * @param {string} userToken -  user token
+     * @param {api-callback} callback -  callback for this call
+     * @throws Will throw an error if userToken is not provided
+     */
+    activateUser(userToken, callback) {
+        if (!userToken) {
+            throw new Error("userToken is required");
+        }
+        const transition = {
+            transition: "ACTIVATED",
+        };
+        this.client.doPost(`users/${encodeURIComponent(userToken)}/status-transitions`, transition, {}, callback);
+    }
+
+    /**
+     * Deactivate a user
+     *
+     * @param {string} userToken -  user token
+     * @param {api-callback} callback -  callback for this call
+     * @throws Will throw an error if userToken is not provided
+     */
+    deactivateUser(userToken, callback) {
+        if (!userToken) {
+            throw new Error("userToken is required");
+        }
+        const transition = {
+            transition: "DE_ACTIVATED",
+        };
+        this.client.doPost(`users/${encodeURIComponent(userToken)}/status-transitions`, transition, {}, callback);
+    }
+
+    /**
+     * Lock a user account
+     *
+     * @param {string} userToken -  user token
+     * @param {api-callback} callback -  callback for this call
+     * @throws Will throw an error if userToken is not provided
+     */
+    lockUser(userToken, callback) {
+        if (!userToken) {
+            throw new Error("userToken is required");
+        }
+        const transition = {
+            transition: "LOCKED",
+        };
+        this.client.doPost(`users/${encodeURIComponent(userToken)}/status-transitions`, transition, {}, callback);
+    }
+    /**
```

Comment:
Pls add a blank line between the methods

Reply:
added black line between the methods. Thanks @arao6